lib/vty.h

```diff
@@ -182,6 +182,11 @@ struct vty_arg
 /* Small macro to determine newline is newline only or linefeed needed. */
 #define VTYNL  ((vty->type == VTY_TERM) ? "\r\n" : "\n")
 
+/* for compatibility */
+#define VTY_NEWLINE                               VTYNL
+#define VTY_GET_INTEGER(desc,v,str)               {(v)=strtoul ((str), NULL, 10);}
```

Comment:
Nobody is using these macros in the code base.

Just on the off chance that you have some patches you need to fix up:

`remove-vty-get.cocci`
```
    @getull@
    expression v;
    expression str;
    @@
    <...
    - VTY_GET_ULL(..., v, str)
    + v = strtoull (str, NULL, 10)
    ...>

    @getul@
    expression v;
    expression str;
    @@
    <...
    - VTY_GET_ULONG(..., v, str)
    + v = strtoul (str, NULL, 10)
    ...>

    @getintrange@
    expression name;
    expression v;
    expression str;
    @@
    <...
    - VTY_GET_INTEGER_RANGE(name, v, str, ...)
    + v = strtoul (str, NULL, 10)
    ...>

    @getint@
    expression v;
    expression str;
    @@
    <...
    - VTY_GET_INTEGER(..., v, str)
    + v = strtoul (str, NULL, 10)
    ...>

    @getv4@
    expression v;
    expression str;
    @@
    <...
    - VTY_GET_IPV4_ADDRESS(..., v, str)
    + inet_aton (str, &v)
    ...>

    @getv4pfx@
    expression v;
    expression str;
    @@
    <...
    - VTY_GET_IPV4_PREFIX(..., v, str)
    + str2prefix_ipv4 (str, &v)
    ...>
```